docs/tutorials/pauli-correlation-encoding-for-qaoa.ipynb

```diff
@@ -7,7 +7,7 @@
    "source": [
     "{/* cspell:ignore setminus coloneqq rbrack  // latex that isn't being ignored for some reason */}\n",
     "\n",
-    "# Solving Maxcut with Reduced Qubit Requirements Using Pauli Correlation Encoding\n",
+    "# Pauli Correlation Encoding to reduce Maxcut requirements\n",
```

Comment:
Does "correlation encoding" need to be upper case? and "maxcut"?

Reply:
correlation encoding can be lower case, but I'm not sure about Maxcut